operating/manage-vulnerabilities/scan-rhcos-node-host.adoc

```diff
@@ -12,18 +12,20 @@ With {rh-rhacs-first}, you can scan RHCOS nodes for vulnerabilities and detect p
 
 {product-title-short} scans RHCOS RPMs installed on the node host, as part of the RHCOS installation, for any known vulnerabilities.
 
-First, {product-title-short} analyzes and detects RHCOS components, and then it matches vulnerabilities for identified components using Red Hat RHEL and OpenShift 4.X Open Vulnerability and Assessment Language (OVAL) v2 security data streams.
+First, {product-title-short} analyzes and detects RHCOS components, and then it matches vulnerabilities for identified components by using Red Hat RHEL and OpenShift 4.X Open Vulnerability and Assessment Language (OVAL) v2 security data streams.
 
 [NOTE]
 ====
+* You must manually enable the RHCOS node scanning features.
```

Comment:
So, technically this is only true for manifest-based installations.
Installations managed by Helm and the Operator will get this feature automatically if OpenShift is detected as a platform.

Reply:
@Maddosaurus OpenShift > 4.7 right?

Reply:
@Maddosaurus we also say requires 4.10 or later at https://github.com/openshift/openshift-docs/pull/59886/files#diff-6458b7d7560d86665cede052a43c9f0e7c571f883f94049b1487410f16a3c0ccR12 Can you confirm the correct OCP version for which we support this feature.

Reply:
The feature works for 4.7 and later, but ACS officially supports 4.10 and later. I would suggest to write 4.10 if in doubt.